docs/source/guide/quickstart.rst

```diff
@@ -98,5 +131,602 @@ Start the Docker container in the example/ directory.
 
     $ docker-compose run --rm --service-ports api
 
-Hit the `http://localhost:5000/v1/characters` url in your browser.  You will see the JSON response
-for the characters endpoint returned.
+
+Fetch a list of Character objects..
+
+.. code-block:: shell
+
+    curl -X GET localhost:5000/v1/characters | python -m json.tool
+
+	{
+		"payload": [
+			{
+				"created_at": "2017-06-04T11:47:02.017094",
+				"id": 1,
+				"name": "Obe Wan"
+			}
+		]
+	}
+
+Add a new Character..
+
+.. code-block:: shell
+
+    curl -H "Content-Type: application/json" -d '{"name":"Darth Vader"}' -X POST localhost:5000/v1/characters | python -m json.tool
+
+	{
+		"payload": [
+			{
+				"created_at": "2017-09-01T04:51:45.456072",
+				"id": 2,
+				"name": "Darth Vader"
+			}
+		]
+	}
+
+
+Fetch a Character by id..
+
+.. code-block:: shell
+
+    curl -X GET localhost:5000/v1/characters/2 | python -m json.tool
+
+	{
+		"payload": {
+            "created_at": "2017-09-01T04:51:45.456072",
+            "id": 2,
+            "name": "Darth Vader"
+		}
+	}
+
+
+Update a Character by id..
+
+.. code-block:: shell
+
+    curl -H "Content-Type: application/json" -d '{"id": 2, "name":"Anakin Skywalker", "created_at": "2017-09-01T04:51:45.456072"}' -X PUT localhost:5000/v1/characters/2 | python -m json.tool
+
+	{
+		"payload": {
+            "created_at": "2017-09-01T04:51:45.456072",
+            "id": 2,
+            "name": "Anakin Skywalker"
+		}
+	}
+
+
+And finally, Delete a Character by id..
+
+.. code-block:: shell
+
+    curl -X DELETE localhost:5000/v1/characters/2
+
+
+URLS && url_for
+^^^^^^^^^^^^^^^^
+
+URLS's are automatically defined by Resoruce's and Endpoint's using Flask's built in url_mapping functionality.  We optionally provide Resource with a url_prefix which is applied to all of its registered Endponts.
```

Comment:
+URLSs are automatically defined by Resources and Endpoints using Flask's built in url_mapping functionality.  We optionally provide Resource with a url_prefix which is applied to all of it's registered Endpoints.